tests/framework/utils/helm_helper.go

```diff
@@ -90,27 +85,3 @@ func (h *HelmHelper) DeleteLocalRookHelmChart(namespace, deployName string) erro
 
 	return nil
 }
-
-func FindRookRoot() (string, error) {
```

Comment:
hmm, maybe we should keep this method for the integration tests. The tests failed that a script was not found:
```
exec: bash: /home/rook/go/src/github.com/rook/rook/tests/scripts/deploy_admission_controller_test.sh: No such file or directory
```

Now I'm remembering that in Jenkins, the full source tree is not available. The `tests/scripts` are in the current dir instead of being in that subdir.

Reply:
What if we changed Jenkins to have the full source tree?

Reply:
NVM, I'm starting to see why it's not possible. The integration tests have a value compiled in which may not be correct if the integration test binary is moved around. I guess we could find a way to compile files into the integration tests to avoid this, but let's not right now.